### ISSUE TITLE
New lang strings added to en.json need to be added to other locales manually.

### ISSUE THREAD

**Issue by rayou** [BoostIO/Boostnote#1719]
Every time a new lang string is introduced, `locales/en.json` will be updated automatically but not the others, currently contributors have to add the new lang strings to the other locales manually. This increases the effort to the development and to code review.

The first suggestion is that create a new task in Travis to check if new strings are added to `locales/en.json`, the rest locales should be updated accordingly.

Second suggestion and I think is the ideal one is to automate this process, we can build a git pre-commit hook if local changes contain a locale file (usually is `locales/en.json`) then compare locale JSONs then add missing keys to those that missed.

**Comment by Rokt33r** [comment]
I agree with you. I assume we should disable `devMode` by default even on development environment. It is quite annoying.
> `devMode` https://github.com/jeresig/i18n-node-2#devmode
